docs/ForDevelopers/Dependencies.md

```diff
@@ -24,45 +29,118 @@ For other recent Linux distributions, you can also rely on the pre-built binarie
 ### Building from Source
 
 ```shell
-wget https://github.com/llvm/llvm-project/releases/download/llvmorg-20.1.0-rc1/llvm-project-20.1.0-rc1.src.tar.xz 
-tar -xf llvm-project-20.1.0-rc1.src.tar.xz 
+wget https://github.com/llvm/llvm-project/releases/download/llvmorg-20.1.0-rc1/llvm-project-20.1.0-rc1.src.tar.xz
+tar -xf llvm-project-20.1.0-rc1.src.tar.xz
 mkdir llvm-project-20.1.0-rc1.src/build
 cd llvm-project-20.1.0-rc1.src
-env VIRTUAL_ENV=/venv cmake -B build  -DLLVM_ENABLE_PROJECTS="llvm;mlir;clang;clang-tools-extra" -DLLVM_TARGETS_TO_BUILD="X86" -DLLVM_BUILD_EXAMPLES=OFF -DCMAKE_BUILD_TYPE=Release -G Ninja -DLLVM_ENABLE_ASSERTIONS=OFF  -DLLVM_BUILD_TESTS=OFF -DLLVM_BUILD_LLVM_DYLIB=ON -DLLVM_LINK_LLVM_DYLIB=OFF -DLLVM_ENABLE_DUMP=ON -DLLVM_ENABLE_FFI=ON -DCMAKE_CXX_FLAGS="-fno-omit-frame-pointer -mno-omit-leaf-frame-pointer" -DLLVM_PARALLEL_LINK_JOBS=1 -DLLVM_PARALLEL_TABLEGEN_JOBS=10 -DBUILD_SHARED_LIBS=OFF -DLLVM_INSTALL_UTILS=ON  -DLLVM_ENABLE_ZLIB=OFF -DCMAKE_INSTALL_PREFIX=[output-dir] llvm/
-RUN  cmake --build build --target install -j$(nproc)
+export INSTALL_PREFIX=[install_prefix]
+env VIRTUAL_ENV=/venv cmake -B build  -DLLVM_ENABLE_PROJECTS="llvm;mlir;clang;clang-tools-extra" -DLLVM_TARGETS_TO_BUILD="X86" -DLLVM_BUILD_EXAMPLES=OFF -DCMAKE_BUILD_TYPE=Release -G Ninja -DLLVM_ENABLE_ASSERTIONS=OFF  -DLLVM_BUILD_TESTS=OFF -DLLVM_BUILD_LLVM_DYLIB=ON -DLLVM_LINK_LLVM_DYLIB=OFF -DLLVM_ENABLE_DUMP=ON -DLLVM_ENABLE_FFI=ON -DCMAKE_CXX_FLAGS="-fno-omit-frame-pointer -mno-omit-leaf-frame-pointer" -DLLVM_PARALLEL_LINK_JOBS=1 -DLLVM_PARALLEL_TABLEGEN_JOBS=10 -DBUILD_SHARED_LIBS=OFF -DLLVM_INSTALL_UTILS=ON  -DLLVM_ENABLE_ZLIB=OFF -DCMAKE_INSTALL_PREFIX=$INSTALL_PREFIX llvm/
+cmake --build build --target install -j$(nproc)
+```
+
+</TabItem>
+<TabItem value="macos" label="MacOS">
+Install LLVM/MLIR using Homebrew:
+
+```shell
+brew install llvm@20
 ```
 
+### Binaries
+⚠️ **Caution**: the pre-built binaries provided by the LLVM project on the Github release pages **DO NOT** serve as a replacement, since they lack the required MLIR support.
+
+### Building from Source
+
+1. Install XCode (through the App Store).
+2. Install the build requisites: `brew install cmake ninja z3`
+2. Make sure to replace [install_prefix] with your preferred install path of LLVM.
+```shell
+wget https://github.com/llvm/llvm-project/releases/download/llvmorg-20.1.4/llvm-project-20.1.4.src.tar.xz
+tar -xf llvm-project-20.1.4.src.tar.xz
+mkdir llvm-project-20.1.4.src/build
+cd llvm-project-20.1.4.src
+export INSTALL_PREFIX=[install_prefix]
+export SDKROOT=$(xcrun --sdk macosx --show-sdk-path)`
```

Comment:
` is missing. Thanks for changing build mode to Release.
export SDKROOT=`$(xcrun --sdk macosx --show-sdk-path)`